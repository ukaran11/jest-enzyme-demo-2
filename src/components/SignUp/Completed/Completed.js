import React from "react";
import styled from "styled-components";

const Message = styled.span`
  font-size: 1.5rem;
`;

const Tick = styled.div`
  font-size: 8rem;
  line-height: 1;
  margin-bottom: 1rem;
`;

const Wrapper = styled.div`
  align-items: center;
  display: flex;
  flex-flow: column nowrap;
  padding: 1rem 0;
`;

const Completed = () => {
  return (
    <Wrapper>
      <Tick>âœ…</Tick>
      <Message>Sign Up Complete!</Message>
    </Wrapper>
  );
};

export default Completed;
