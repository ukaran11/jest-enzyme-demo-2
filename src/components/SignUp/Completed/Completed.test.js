import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";

import Completed from "./Completed";

test("matched snapshot", () => {
    const wrapper = shallow(<Completed />);

    expect(toJSON(wrapper)).toMatchSnapshot();
})