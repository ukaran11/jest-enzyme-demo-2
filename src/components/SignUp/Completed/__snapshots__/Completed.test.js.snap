// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matched snapshot 1`] = `
<styled.div>
  <styled.div>
    âœ…
  </styled.div>
  <styled.span>
    Sign Up Complete!
  </styled.span>
</styled.div>
`;
